DAOS-12092 test: Move HW Large tests to HW Medium

phender opened the pull request:
Moving hw,large tagged tests to hw,medium to reduce cluster complexity.

Test-tags: MultipleContainerDelete SnapshotAggregation daos_perf test_daos_racer test_daos_racer_simple DmvrObjLargePosix DmvrSerialLargePosix deployment test_pool_services IoConsistency LargeFileCount MacsioTest SmallFileCount DaosRunIoConf

Required-githooks: true

Signed-off-by: Phil Henderson <phillip.henderson@intel.com>

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficent testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/1/execution/node/1138/log

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/4/execution/node/718/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/3/execution/node/750/log

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/7/execution/node/670/log

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 commented:
Test stage Functional Hardware Medium Verbs Provider completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/9/execution/node/677/log

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/9/execution/node/771/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/9/execution/node/724/log

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

phender commented:
*** Updated PR & description ***

DAOS-12092 test: Move HW Large tests to HW Medium

Replacing hw,large test tags with hw,medium were applicable.

Signed-off-by: Phil Henderson <phillip.henderson@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Medium Verbs Provider completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/10/execution/node/1083/log

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/10/execution/node/1104/log

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/11/execution/node/1058/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/11/execution/node/1144/log

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/13/execution/node/407/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/13/execution/node/428/log

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/14/execution/node/950/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/14/execution/node/1018/log

phender commented:
Test Failures in https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/14/:

- Test timeout during the end of tearDown; probably need to increase this as we increased the pool size in this PR
  - [Test Hardware / Functional Hardware Medium / FTEST_datamover.DmvrSerialLargePosix.1-./datamover/serial_large_posix.py:DmvrSerialLargePosix.test_dm_serial_large_posix_dserialize;run-container-ddeserialize-client_processes-dfuse-dserialize-hosts-mdtest-pool-server_config-engines-0-storage-0-1-5e23](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/14/testReport/junit/FTEST_datamover/DmvrSerialLargePosix/Test_Hardware___Functional_Hardware_Medium___1___datamover_serial_large_posix_py_DmvrSerialLargePosix_test_dm_serial_large_posix_dserialize_run_container_ddeserialize_client_processes_dfuse_dserialize_hosts_mdtest_pool_server_config_engines_0_storage_0_1_5e23/)

- Added https://daosio.atlassian.net/browse/DAOS-12235 and corresponding master skip-list entry for:
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.01-./daos_test/suite.py:DaosCoreTest.test_daos_degraded_mode;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-6927](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/14/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___01___daos_test_suite_py_DaosCoreTest_test_daos_degraded_mode_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_6927/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DAOS_Degraded-mode.DEGRADED1: Degraded mode during updates](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/14/testReport/junit/FTEST_daos_test/DAOS_Degraded-mode/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___DEGRADED1__Degraded_mode_during_updates/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DAOS_Degraded-mode.DEGRADED2: Degraded mode during lookup](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/14/testReport/junit/FTEST_daos_test/DAOS_Degraded-mode/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___DEGRADED2__Degraded_mode_during_lookup/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DAOS_Degraded-mode.DEGRADED3: Degraded mode during enumerate](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/14/testReport/junit/FTEST_daos_test/DAOS_Degraded-mode/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___DEGRADED3__Degraded_mode_during_enumerate/)

- Added https://daosio.atlassian.net/browse/DAOS-12236 and corresponding master skip-list entry for:
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.23-./daos_test/suite.py:DaosCoreTest.test_daos_rebuild_ec;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-6927](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/14/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___23___daos_test_suite_py_DaosCoreTest_test_daos_rebuild_ec_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_6927/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DAOS_IO_EC_4P2G1.IO31: fetch when all replicas unavailable](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/14/testReport/junit/FTEST_daos_test/DAOS_IO_EC_4P2G1/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___IO31__fetch_when_all_replicas_unavailable/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DAOS_Rebuild_EC.23-./daos_test/suite.py:DaosCoreTest.test_daos_rebuild_ec;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-6927](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/14/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___23___daos_test_suite_py_DaosCoreTest_test_daos_rebuild_ec_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_6927/)


 - Applying latest master changes to see if these reproduce:

   - [Test Hardware / Functional Hardware Medium / FTEST_osa.OSAOfflineExtend.4-./osa/offline_extend.py:OSAOfflineExtend.test_osa_offline_extend_oclass;run-aggregation-checksum-container-extra_servers-hosts-ior-clientslots-iorflags-loop_test-mdtest-client_processes-wr_size-32K-pool-rebuild-server_config-engines-0-storage-0-1-setup-snapshot-test_obj_class-test_ranks-c712](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/14/testReport/junit/FTEST_osa/OSAOfflineExtend/Test_Hardware___Functional_Hardware_Medium___4___osa_offline_extend_py_OSAOfflineExtend_test_osa_offline_extend_oclass_run_aggregation_checksum_container_extra_servers_hosts_ior_clientslots_iorflags_loop_test_mdtest_client_processes_wr_size_32K_pool_rebuild_server_config_engines_0_storage_0_1_setup_snapshot_test_obj_class_test_ranks_c712/)
   - [Test Hardware / Functional Hardware Medium / FTEST_osa.OSAOfflineExtend.5-./osa/offline_extend.py:OSAOfflineExtend.test_osa_offline_extend_during_aggregation;run-aggregation-checksum-container-extra_servers-hosts-ior-clientslots-iorflags-loop_test-mdtest-client_processes-wr_size-32K-pool-rebuild-server_config-engines-0-storage-0-1-setup-snapshot-test_obj_class-test_ranks-c712](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/14/testReport/junit/FTEST_osa/OSAOfflineExtend/Test_Hardware___Functional_Hardware_Medium___5___osa_offline_extend_py_OSAOfflineExtend_test_osa_offline_extend_during_aggregation_run_aggregation_checksum_container_extra_servers_hosts_ior_clientslots_iorflags_loop_test_mdtest_client_processes_wr_size_32K_pool_rebuild_server_config_engines_0_storage_0_1_setup_snapshot_test_obj_class_test_ranks_c712/)

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/15/execution/node/1057/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/15/execution/node/1010/log

phender commented:
Test failures in 

- New issue https://daosio.atlassian.net/browse/DAOS-12244
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.18-./daos_test/suite.py:DaosCoreTest.test_daos_rebuild_simple;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-6927](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/15/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___18___daos_test_suite_py_DaosCoreTest_test_daos_rebuild_simple_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_6927/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DAOS_Rebuild_Simple.18-./daos_test/suite.py:DaosCoreTest.test_daos_rebuild_simple;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-6927](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/15/testReport/junit/FTEST_daos_test/DAOS_Rebuild_Simple/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___18___daos_test_suite_py_DaosCoreTest_test_daos_rebuild_simple_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_6927/)

- New issue https://daosio.atlassian.net/browse/DAOS-12241
  - [Test Hardware / Functional Hardware Medium / FTEST_io.LargeFileCount.1-./io/large_file_count.py:LargeFileCount.test_largefilecount;run-container-dfuse-hosts-ior-largefilecount-mdtest-client_processes-pool-server_config-engines-0-storage-0-1-c5dc](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/15/testReport/junit/FTEST_io/LargeFileCount/Test_Hardware___Functional_Hardware_Medium___1___io_large_file_count_py_LargeFileCount_test_largefilecount_run_container_dfuse_hosts_ior_largefilecount_mdtest_client_processes_pool_server_config_engines_0_storage_0_1_c5dc/)

- New issue https://daosio.atlassian.net/browse/DAOS-12242
  - [Test Hardware / Functional Hardware Medium / FTEST_osa.OSAOfflineExtend.4-./osa/offline_extend.py:OSAOfflineExtend.test_osa_offline_extend_oclass;run-aggregation-checksum-container-extra_servers-hosts-ior-clientslots-iorflags-loop_test-mdtest-client_processes-wr_size-32K-pool-rebuild-server_config-engines-0-storage-0-1-setup-snapshot-test_obj_class-test_ranks-73fd](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/15/testReport/junit/FTEST_osa/OSAOfflineExtend/Test_Hardware___Functional_Hardware_Medium___4___osa_offline_extend_py_OSAOfflineExtend_test_osa_offline_extend_oclass_run_aggregation_checksum_container_extra_servers_hosts_ior_clientslots_iorflags_loop_test_mdtest_client_processes_wr_size_32K_pool_rebuild_server_config_engines_0_storage_0_1_setup_snapshot_test_obj_class_test_ranks_73fd/)


- New issue https://daosio.atlassian.net/browse/DAOS-12243
  - [Test Hardware / Functional Hardware Medium / FTEST_osa.OSAOfflineExtend.5-./osa/offline_extend.py:OSAOfflineExtend.test_osa_offline_extend_during_aggregation;run-aggregation-checksum-container-extra_servers-hosts-ior-clientslots-iorflags-loop_test-mdtest-client_processes-wr_size-32K-pool-rebuild-server_config-engines-0-storage-0-1-setup-snapshot-test_obj_class-test_ranks-73fd](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/15/testReport/junit/FTEST_osa/OSAOfflineExtend/Test_Hardware___Functional_Hardware_Medium___5___osa_offline_extend_py_OSAOfflineExtend_test_osa_offline_extend_during_aggregation_run_aggregation_checksum_container_extra_servers_hosts_ior_clientslots_iorflags_loop_test_mdtest_client_processes_wr_size_32K_pool_rebuild_server_config_engines_0_storage_0_1_setup_snapshot_test_obj_class_test_ranks_73fd/)

- Known issue https://daosio.atlassian.net/browse/DAOS-11996
  - [Test Hardware / Functional Hardware Medium / FTEST_rebuild.RbldContainerCreate.1-./rebuild/container_create_race.py:RbldContainerCreate.test_rebuild_container_create;run-container-hosts-io-ior-client_processes-transfersize_blocksize-1M-pool-server_config-engines-0-storage-0-1-testparams-ranks-cc08](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/15/testReport/junit/FTEST_rebuild/RbldContainerCreate/Test_Hardware___Functional_Hardware_Medium___1___rebuild_container_create_race_py_RbldContainerCreate_test_rebuild_container_create_run_container_hosts_io_ior_client_processes_transfersize_blocksize_1M_pool_server_config_engines_0_storage_0_1_testparams_ranks_cc08/)

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

phender commented:
Last push was required to resolve conflict with Jenkinsfile.

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/16/execution/node/1273/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/16/execution/node/1226/log

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daltonbohning commented on src/tests/ftest/aggregation/basic.yaml:
```diff
@@ -28,8 +28,7 @@ ior:
   client_processes:
-    np_12:
-      np: 12
+    ppn: 6
```
Since the number of servers is the same, we could maybe keep the same number of total client processes. Though I'm not sure if it matters for this test.

daltonbohning commented on src/tests/ftest/control/config_generate_output.py:
```diff
@@ -429,7 +426,7 @@ def test_net_class(self):
         :avocado: tags=all,full_regression
         :avocado: tags=hw,large
-        :avocado: tags=control,config_generate_entries
+        :avocado: tags=control,
```
```suggestion
        :avocado: tags=control
```

phender replied:
Fixed

daltonbohning commented on src/tests/ftest/server/storage_tiers.yaml:
```diff
@@ -1,50 +1,134 @@
   test_clients: 1
+
+setup:
+  start_servers: False
+  start_agents: false
```
```suggestion
  start_servers: false
  start_agents: false
```

phender replied:
Fixed

daosbuild1 commented:
Test stage Functional Hardware Medium UCX Provider completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/17/execution/node/1187/log

daosbuild1 commented:
Test stage Functional Hardware Medium Verbs Provider completed with status UNSTABLE.  https://build.hpdd.intel.com/job/daos-stack/job/daos//view/change-requests/job/PR-10718/17/testReport/(root)/

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/17/execution/node/1166/log

phender commented:
Failures in https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17:

- Known issue https://daosio.atlassian.net/browse/DAOS-12254
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.01-./daos_test/suite.py:DaosCoreTest.test_daos_degraded_mode;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___01___daos_test_suite_py_DaosCoreTest_test_daos_degraded_mode_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.02-./daos_test/suite.py:DaosCoreTest.test_daos_management;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___02___daos_test_suite_py_DaosCoreTest_test_daos_management_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.03-./daos_test/suite.py:DaosCoreTest.test_daos_pool;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___03___daos_test_suite_py_DaosCoreTest_test_daos_pool_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.04-./daos_test/suite.py:DaosCoreTest.test_daos_container;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___04___daos_test_suite_py_DaosCoreTest_test_daos_container_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.05-./daos_test/suite.py:DaosCoreTest.test_daos_epoch;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___05___daos_test_suite_py_DaosCoreTest_test_daos_epoch_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.06-./daos_test/suite.py:DaosCoreTest.test_daos_single_rdg_tx;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___06___daos_test_suite_py_DaosCoreTest_test_daos_single_rdg_tx_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.07-./daos_test/suite.py:DaosCoreTest.test_daos_distributed_tx;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___07___daos_test_suite_py_DaosCoreTest_test_daos_distributed_tx_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.08-./daos_test/suite.py:DaosCoreTest.test_daos_verify_consistency;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___08___daos_test_suite_py_DaosCoreTest_test_daos_verify_consistency_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.09-./daos_test/suite.py:DaosCoreTest.test_daos_io;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___09___daos_test_suite_py_DaosCoreTest_test_daos_io_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.10-./daos_test/suite.py:DaosCoreTest.test_daos_ec_io;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___10___daos_test_suite_py_DaosCoreTest_test_daos_ec_io_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.11-./daos_test/suite.py:DaosCoreTest.test_daos_ec_obj;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___11___daos_test_suite_py_DaosCoreTest_test_daos_ec_obj_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.12-./daos_test/suite.py:DaosCoreTest.test_daos_object_array;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___12___daos_test_suite_py_DaosCoreTest_test_daos_object_array_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.13-./daos_test/suite.py:DaosCoreTest.test_daos_array;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___13___daos_test_suite_py_DaosCoreTest_test_daos_array_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.14-./daos_test/suite.py:DaosCoreTest.test_daos_kv;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___14___daos_test_suite_py_DaosCoreTest_test_daos_kv_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.15-./daos_test/suite.py:DaosCoreTest.test_daos_capability;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___15___daos_test_suite_py_DaosCoreTest_test_daos_capability_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.16-./daos_test/suite.py:DaosCoreTest.test_daos_epoch_recovery;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___16___daos_test_suite_py_DaosCoreTest_test_daos_epoch_recovery_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.17-./daos_test/suite.py:DaosCoreTest.test_daos_md_replication;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___17___daos_test_suite_py_DaosCoreTest_test_daos_md_replication_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.19-./daos_test/suite.py:DaosCoreTest.test_daos_drain_simple;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___19___daos_test_suite_py_DaosCoreTest_test_daos_drain_simple_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.20-./daos_test/suite.py:DaosCoreTest.test_daos_extend_simple;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___20___daos_test_suite_py_DaosCoreTest_test_daos_extend_simple_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.21-./daos_test/suite.py:DaosCoreTest.test_daos_oid_allocator;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___21___daos_test_suite_py_DaosCoreTest_test_daos_oid_allocator_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.22-./daos_test/suite.py:DaosCoreTest.test_daos_checksum;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___22___daos_test_suite_py_DaosCoreTest_test_daos_checksum_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.24-./daos_test/suite.py:DaosCoreTest.test_daos_aggregate_ec;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___24___daos_test_suite_py_DaosCoreTest_test_daos_aggregate_ec_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.25-./daos_test/suite.py:DaosCoreTest.test_daos_degraded_ec;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___25___daos_test_suite_py_DaosCoreTest_test_daos_degraded_ec_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)
  - [Test Hardware / Functional Hardware Medium UCX Provider / FTEST_daos_test.DaosCoreTest.26-./daos_test/suite.py:DaosCoreTest.test_daos_dedup;run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-engines-0-storage-0-1-timeouts-fa5e](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DaosCoreTest/Test_Hardware___Functional_Hardware_Medium_UCX_Provider___26___daos_test_suite_py_DaosCoreTest_test_daos_dedup_run_agent_config_transport_config_daos_tests_args_daos_test_num_clients_pools_created_scalable_endpoint_stopped_ranks_test_name_dmg_hosts_pool_server_config_engines_0_storage_0_1_timeouts_fa5e/)

- Known issue https://daosio.atlassian.net/browse/DAOS-8714
  - [Test Hardware / Functional Hardware Medium / FTEST_daos_vol.DaosVol.2-./daos_vol/h5_suite.py:DaosVol.test_daos_vol_mpich;run-container-daos_vol_tests-test2-dfuse-hosts-job_manager-pool-server_config-engines-0-storage-0-1-5244](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_vol/DaosVol/Test_Hardware___Functional_Hardware_Medium___2___daos_vol_h5_suite_py_DaosVol_test_daos_vol_mpich_run_container_daos_vol_tests_test2_dfuse_hosts_job_manager_pool_server_config_engines_0_storage_0_1_5244/)
  - [Test Hardware / Functional Hardware Medium / FTEST_daos_vol.DaosVol.3-./daos_vol/h5_suite.py:DaosVol.test_daos_vol_mpich;run-container-daos_vol_tests-test3-dfuse-hosts-job_manager-pool-server_config-engines-0-storage-0-1-1330](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_vol/DaosVol/Test_Hardware___Functional_Hardware_Medium___3___daos_vol_h5_suite_py_DaosVol_test_daos_vol_mpich_run_container_daos_vol_tests_test3_dfuse_hosts_job_manager_pool_server_config_engines_0_storage_0_1_1330/)

- New issue: test timeout (stage ran with tcp)
  - [Test Hardware / Functional Hardware Medium / FTEST_ior.IorSmall.1-./ior/small.py:IorSmall.test_ior_small;run-agent_config-transport_config-container-dfuse-dmg-hdf5_vol-hosts-ior-client_processes-iorflags-pool-server_config-engines-0-storage-0-1-9acc](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_ior/IorSmall/Test_Hardware___Functional_Hardware_Medium___1___ior_small_py_IorSmall_test_ior_small_run_agent_config_transport_config_container_dfuse_dmg_hdf5_vol_hosts_ior_client_processes_iorflags_pool_server_config_engines_0_storage_0_1_9acc/)

- New issue: one test teardown error and the rest are segmentation faults
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD11: rebuild2p partial update with parity tgt fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD11__rebuild2p_partial_update_with_parity_tgt_fail/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD14: rebuild dfs io with data(s0) tgt fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD14__rebuild_dfs_io_with_data_s0__tgt_fail_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD15: rebuild dfs io with data(s1) tgt fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD15__rebuild_dfs_io_with_data_s1__tgt_fail_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD16: rebuild dfs io with data(s1) tgt fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD16__rebuild_dfs_io_with_data_s1__tgt_fail_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD17: rebuild dfs io with data(s0, s1) tgt fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD17__rebuild_dfs_io_with_data_s0__s1__tgt_fail_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD18: rebuild dfs io with data(s0, s2) tgt fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD18__rebuild_dfs_io_with_data_s0__s2__tgt_fail_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD19: rebuild dfs io with data(s0, s3) tgt fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD19__rebuild_dfs_io_with_data_s0__s3__tgt_fail_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD20: rebuild dfs io with data(s1, s2) tgt fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD20__rebuild_dfs_io_with_data_s1__s2__tgt_fail_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD21: rebuild dfs io with data(s1, s3) tgt fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD21__rebuild_dfs_io_with_data_s1__s3__tgt_fail_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD22: rebuild dfs io with data(s2, s3) tgt fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD22__rebuild_dfs_io_with_data_s2__s3__tgt_fail_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD23: rebuild dfs io with 1data 1parity(s0, p1)](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD23__rebuild_dfs_io_with_1data_1parity_s0__p1_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD24: rebuild dfs io with 1data 1parity(s3, p1)](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD24__rebuild_dfs_io_with_1data_1parity_s3__p1_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD25: rebuild dfs io with 1data 1parity(s2, p1)](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD25__rebuild_dfs_io_with_1data_1parity_s2__p1_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD26: rebuild dfs io with 1data 1parity(s0, p0)](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD26__rebuild_dfs_io_with_1data_1parity_s0__p0_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD27: rebuild dfs io with 1data 1parity(s3, p0)](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD27__rebuild_dfs_io_with_1data_1parity_s3__p0_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD28: rebuild dfs io with 1data 1parity(s2, p0)](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD28__rebuild_dfs_io_with_1data_1parity_s2__p0_/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD29: rebuild dfs io with sequential data(s0, s1) fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD29__rebuild_dfs_io_with_sequential_data_s0__s1__fail/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD30: rebuild dfs io with sequential data(s1, s2) fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD30__rebuild_dfs_io_with_sequential_data_s1__s2__fail/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD31: rebuild dfs io with sequential data(s2, s3) fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD31__rebuild_dfs_io_with_sequential_data_s2__s3__fail/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD32: rebuild dfs io with sequential data(s0, s3) fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD32__rebuild_dfs_io_with_sequential_data_s0__s3__fail/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD33: rebuild dfs io with data and parity(s0, p0) fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD33__rebuild_dfs_io_with_data_and_parity_s0__p0__fail/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD34: rebuild dfs io with data and parity(s3, p1) fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD34__rebuild_dfs_io_with_data_and_parity_s3__p1__fail/)
  - [Test Hardware / Functional Hardware Medium Verbs Provider / FTEST_daos_test.DAOS_Rebuild_EC.REBUILD35: rebuild dfs io with 2 parities(p0, p1) fail](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/17/testReport/junit/FTEST_daos_test/DAOS_Rebuild_EC/Test_Hardware___Functional_Hardware_Medium_Verbs_Provider___REBUILD35__rebuild_dfs_io_with_2_parities_p0__p1__fail/)

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 reviewed (commented):
LGTM.  No errors found by checkpatch.

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-10718/19/execution/node/1214/log

phender commented:
Failures in https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/19/:

- Known issue https://daosio.atlassian.net/browse/DAOS-8714
  - [Test Hardware / Functional Hardware Medium / FTEST_daos_vol.DaosVol.2-./daos_vol/h5_suite.py:DaosVol.test_daos_vol_mpich;run-container-daos_vol_tests-test2-dfuse-hosts-job_manager-pool-server_config-engines-0-storage-0-1-5244](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/19/testReport/junit/FTEST_daos_vol/DaosVol/Test_Hardware___Functional_Hardware_Medium___2___daos_vol_h5_suite_py_DaosVol_test_daos_vol_mpich_run_container_daos_vol_tests_test2_dfuse_hosts_job_manager_pool_server_config_engines_0_storage_0_1_5244/)
  - [Test Hardware / Functional Hardware Medium / FTEST_daos_vol.DaosVol.3-./daos_vol/h5_suite.py:DaosVol.test_daos_vol_mpich;run-container-daos_vol_tests-test3-dfuse-hosts-job_manager-pool-server_config-engines-0-storage-0-1-1330](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10718/19/testReport/junit/FTEST_daos_vol/DaosVol/Test_Hardware___Functional_Hardware_Medium___3___daos_vol_h5_suite_py_DaosVol_test_daos_vol_mpich_run_container_daos_vol_tests_test3_dfuse_hosts_job_manager_pool_server_config_engines_0_storage_0_1_1330/)
